deptrac/stubs/Override.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+//stub Override attribute for PHP <8.3
+
+namespace {
+    if (!class_exists('Override')) {
+        #[\Attribute(\Attribute::TARGET_METHOD)]
+        final class Override {}
```

Comment:
Can this be removed as we already require `symfony/polyfill-php83`?

Reply:
Yes, it can. I had to be quite specific about paths, as deptrac tries to analyze the polyfills, and have just included polyfill-php83 for now. Also renamed stubs to polyfills for clarity.